symfony/plugins/orangehrmCorePlugin/Utility/MailTransport.php

```diff
@@ -19,9 +19,68 @@
 
 namespace OrangeHRM\Core\Utility;
 
-use Swift_SmtpTransport;
+use Symfony\Component\Mailer\Exception\TransportExceptionInterface;
+use Symfony\Component\Mailer\SentMessage;
+use Symfony\Component\Mailer\Transport\Smtp\EsmtpTransport;
+use Symfony\Component\Mailer\Transport\AbstractTransport;
+use Symfony\Component\Mailer\Transport\SendmailTransport;
 
-class MailTransport extends Swift_SmtpTransport
+class MailTransport extends AbstractTransport
 {
+    /**
+     * @var AbstractTransport|SendmailTransport|EsmtpTransport
+     */
+    private AbstractTransport $mailTransport;
 
+    /**
+     * MailTransport constructor.
+     * @param string $host
+     * @param int $port
+     */
+    public function __construct(string $host = 'localhost', int $port = 0)
```

Comment:
not sure `int $port = 0` or `?int $port = null` is best approach

Reply:
Yes. Null is better.

Reply:
Yes. Null is better.